IBPSA/Resources/Scripts/travis/usr/local/bin/jm_ipython.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+IMG_NAME=ubuntu-1604_jmodelica_trunk
+DOCKER_USERNAME=michaelwetter
+# Export the MODELICAPATH
+if [ -z ${MODELICAPATH+x} ]; then
+    MODELICAPATH=`pwd`
+else
+    # Add the current directory to the front of the Modelica path.
+    # This will export the directory to the docker, and also set
+    # it in the MODELICAPATH so that JModelica finds it.
+    MODELICAPATH=`pwd`:${MODELICAPATH}
+fi
+
+# Each entry in the MODELICAPATH will be a mounted read-only volume
+MOD_MOUNT=""
+for ele in ${MODELICAPATH//:/ }; do
+    MOD_MOUNT="${MOD_MOUNT} -v ${ele}:/mnt${ele}:ro"
+done
+
+ # On Darwin, the exported temporary folder needs to be /private/var/folders, not /var/folders
+# see https://askubuntu.com/questions/600018/how-to-display-the-paths-in-path-separately
+if [ `uname` == "Darwin" ]; then
+    MOD_MOUNT=`echo ${MOD_MOUNT} | sed -e 's| /var/folders/| /private/var/folders/|g'`
+fi
+
+# Prepend /mnt/ in front of each entry, which will then be used as the
+DOCKER_MODELICAPATH=`(set -f; IFS=:; printf "/mnt%s:" ${MODELICAPATH})`
+# Cut the trailing ':'
+DOCKER_MODELICAPATH=${DOCKER_MODELICAPATH%?}
+
+cur_dir=`pwd`
+bas_nam=`basename ${cur_dir}`
+sha_dir=`dirname ${cur_dir}`
+# If the current directory is part of the argument list,
+# replace it with . as the docker may have a different file structure
+arg_lis=`echo $@ | sed -e "s|${cur_dir}|.|g"`
+
+docker run \
+	--detach=false \
+    ${MOD_MOUNT} \
+    -v ${sha_dir}:/mnt/shared \
+	-e DISPLAY=${DISPLAY} \
```

Comment:
do you need this line for JModelica?

Reply:
@Mathadon For travis and JModelica, the DISPLAY is probably not needed. But if present, the same script can be used to run JModelica locally in a docker and plot results from the python script invoked by jm_ipython.py. Therefore, rather than having two separate scripts, I prefer to keep the assignment.